### PR TITLE
Simple Titles on Multi Menu

### PR DESCRIPTION
**Pull request by scott-massey**
### Feature
This PR adds support for a simple title for MultiMenus. If a menu item has the prop `type` set to `"title"`, then it'll render the item as a title with different styling.

### CODE REVIEW

**Comment by scott-massey** [comment]
[Magic text in external collections](https://trello.com/c/V21xBysv/2341-magic-text-in-external-collections)

**Review by zachbharris — APPROVED** [review]

File: lib/index.js [hunk 1]
```diff
@@ -288,9 +288,7 @@ var MultiMenuWrapper = exports.MultiMenuWrapper = function (_Component2) {
           className: (0, _classnames2.default)('multi-menu-wrapper', 'expand-' + expandDirection, 'expand-' + verticalExpand, className, { 'multi-menu-wrapper-attached': isStyledMenu }),
           style: position
         },
-        _react2.default.createElement(_reactDocumentEvents2.default, {
```
Comment (nucks) [inline lib/index.js:291]: It doesn't look like the gitignore is working on lib files like you expected it to.

File: src/index.js [hunk 2]
```diff
@@ -263,12 +265,18 @@ export class MultiMenu extends Component {
       openIndex = openPath[basePath.length]
     }
 
-    if (openIndex === undefined || nested) { return null }
-    if (menu[openIndex] && menu[openIndex].inline) { return null }
+    if (openIndex === undefined || nested) {
+      return null
+    }
```
Comment (nucks) [inline src/index.js:270]:
```suggestion
    }

```

File: src/index.js [hunk 3]
```diff
@@ -569,40 +598,40 @@ export class MenuItem extends Component {
...
+        disabled = true
+      } else if (data.type === 'title') {
+        disabled = true
+      }
     }
 
     let title = typeof data.label === 'string' ? data.label : undefined
     let children = this.getChildren()
 
+    console.log(data, 'is title?', data.type === 'title')
```
Comment (nucks) [inline src/index.js:613]:
```suggestion
```

File: lib/index.js (hunk 1 above)
Comment (scott-massey) [reply lib/index.js:291]: Yeah, not totally sure here. I tried to add the folder to .gitignore but wasn't successful. I don't think it was ever on there.

Comment (zachbharris) [reply lib/index.js:291]: Try this @scott-massey https://stackoverflow.com/a/30360954

Comment (scott-massey) [reply lib/index.js:291]: Done

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+name: Deploy Production
+
+on:
+  push:
+    branches:
+      - master
```
Comment (zachbharris) [inline .github/workflows/deploy.yml:6]: This should be main, correct?

```suggestion
      - main
```

**Review by zachbharris — CHANGES_REQUESTED** [review]
Just need to update the `deploy.yml` file and should be good :)

File: .github/workflows/deploy.yml (hunk 4 above)
Comment (scott-massey) [reply .github/workflows/deploy.yml]: Oh, whoops! Fixed that. I copied the deploy script from constants and forgot to change that.

**Review by zachbharris — APPROVED** [review]

**Review by nucks — APPROVED** [review]

**Pull request merged**